src/component/BpmnVisualization.ts

```diff
@@ -27,7 +27,7 @@ import { htmlElement } from './helpers/dom-utils';
 /**
  * @category Initialization
  */
-export default class BpmnVisualization {
```

Comment:
There is a reason to remove `default` ?

Reply:
yes see PR description, this makes reexport harder to do.